man/systemd-vmspawn.xml

```diff
@@ -497,6 +497,48 @@ $ mkosi -d arch -p systemd -p linux --autologin -o image.raw -f build
 $ systemd-vmspawn --image=image.raw
       </programlisting>
     </example>
+
+    <example>
+      <title>Import and run a Fedora 39 Cloud image using machinectl</title>
+
+      <programlisting>
+$ wget https://download.fedoraproject.org/pub/fedora/linux/releases/39/Cloud/x86_64/images/Fedora-Cloud-Base-39-1.5.x86_64.raw.xz \
+       https://download.fedoraproject.org/pub/fedora/linux/releases/39/Cloud/x86_64/images/Fedora-Cloud-39-1.5-x86_64-CHECKSUM
+$ curl -O https://fedoraproject.org/fedora.gpg
+$ gpgv --keyring ./fedora.gpg Fedora-Cloud-39-1.5-x86_64-CHECKSUM
+$ sha256sum -c Fedora-Cloud-39-1.5-x86_64-CHECKSUM
+# machinectl import-raw Fedora-Cloud-Base-39-1.5.x86_64.raw.xz fedora-39-cloud
+# systemd-vmspawn -M fedora-39-cloud
+      </programlisting>
+    </example>
+
+    <example>
+      <title>Build and run systemd's system image and forward the VM's journal to a local file</title>
+
+      <programlisting>
+$ mkosi build
+$ systemd-vmspawn \
+    -D mkosi.output/system \
+    --private-users $(grep $(whoami) /etc/subuid | cut -d: -f2) \
+    --linux mkosi.output/system.efi \
+    --forward-journal=vm.journal \
+    enforcing=0
+      </programlisting>
+    </example>
+
+    <example>
+      <title>SSH into a running VM using the systemd-ssh-proxy</title>
+
+      <programlisting>
+$ mkosi build
+$ systemd-vmspawn \
+    -D mkosi.output/system \
+    --private-users $(grep $(whoami) /etc/subuid | cut -d: -f2)
+    --linux mkosi.output/system.efi \
+    enforcing=0
+$ ssh root@vsock/681331013 -i /run/user/$UID/systemd/vmspawn/machine-*-system-ed2551
```

Comment:
If we're going to be sshing in by CID and vmspawn doesn't directly report the CID, I think we need to pick one arbitrarily to set on the vmspawn command-line.

Reply:
This is more of a temporary solution while machinectl shell is still in development, though I don't have anything against adding a --cid= option?
Just realised I have also typo'd on this line, ed2551->ed25519.

Reply:
Wait this already exists... I'll update the example to set the CID.

Reply:
I've now used --vsock-cid= to set the CID for this example.